Source/Renderer/Context.js

```diff
@@ -822,6 +837,14 @@ define([
         if (count < 0) {
             throw new DeveloperError('drawCommand.count must be omitted or greater than or equal to zero.');
         }
+
+        if (instanceCount < 1) {
```

Comment:
Nevermind, just leave the logic as is, but change the wording of the `DeveloperError` so that `instanceCount` is required, e.g., just say "drawCommand.instanceCount must be greater than or equal to one."